docs/src/operations/upgrade.md

```diff
@@ -0,0 +1,152 @@
+# Upgrade Zeebe
+
+This section describes how to upgrade Zeebe to a new version.
+
+> **Important:** Currently, we are facing an issue that can corrupt the data when upgrading to a new version. The issue affects the reprocessing (i.e. rehydrating the data from the records on the log stream) and can be omitted by restoring the data from a snapshot. Please follow the recommended procedure to minimize the risk of losing data.
```

Comment:
Can we link to a specific issue here?

Reply:
You're not the first one asking this :sweat_smile: We have no explicit issue yet. I'll create one and link it here.

Reply:
So, here is the issue: #5581